packages/mui-material/src/zero-styled/index.tsx

```diff
@@ -2,15 +2,18 @@ import * as React from 'react';
 import { extendSxProp } from '@mui/system/styleFunctionSx';
 import useThemeProps from '../styles/useThemeProps';
 import useTheme from '../styles/useTheme';
-import GlobalStyles, { GlobalStylesProps } from '../GlobalStyles';
+import GlobalStyles from '../GlobalStyles';
 
 export { css, keyframes } from '@mui/system';
 
 export { default as styled } from '../styles/styled';
 
-export function globalCss(styles: GlobalStylesProps['styles']) {
+export function globalCss(styles: any) {
```

Comment:
Could this be

```suggestion
export function globalCss(styles: GlobalStylesProps['styles'] | (theme: Theme) => GlobalStylesProps['styles']) {
```

Reply:
That does not work with array because `GlobalStylesProps['styles']` contains union of primitive, array of primitive and function, function.

I got it working by importing `Interpolation` from MUI system instead.